### DISCUSSION TITLE
Media Manager Catagory Types

### DISCUSSION THREAD

**Discussion by LaocheXe** [e107inc/e107#5050] [Q&A Core] [answered]
Currently, Media Manager Categories have three types listed, images, files, and videos. Could we make it so we could select Images, and Videos, or maybe add Media which would allow both images, and videos (such as youtube, etc...)?

**Comment by tgtje** [comment, marked as answer]
@LaocheXe  Your question is too general, please specify what you exactly seek. 
Is it selection mode when adding a category?**  or 'bundle' image, file and video to 1 select category (Yes)  , or the filter (1 or all) etc...

** likely no, as each type has its own container in storage.
